worth2/templates/base.html

```diff
@@ -55,27 +58,31 @@
     </ul>
 
     <ul class="nav navbar-nav navbar-right">
-        {% if request.user.profile.is_participant %}
+        {% if user.profile.is_participant %}
```

Comment:
When displaying this template to an anonymous user, the `user` variable here will be an instance of `AnonymousUser`: https://docs.djangoproject.com/en/1.7/topics/auth/default/#authentication-data-in-templates so it seems to be fine.